test_typing_inspect.py

```diff
@@ -165,39 +237,71 @@ def test_last_args(self):
         S = TypeVar('S')
         self.assertEqual(get_last_args(int), ())
         self.assertEqual(get_last_args(Union), ())
-        self.assertEqual(get_last_args(ClassVar[int]), (int,))
+        if WITH_CLASSVAR:
+            self.assertEqual(get_last_args(ClassVar[int]), (int,))
         self.assertEqual(get_last_args(Union[T, int]), (T, int))
-        self.assertEqual(get_last_args(Iterable[Tuple[T, S]][int, T]), (int, T))
-        self.assertEqual(get_last_args(Callable[[T, S], int]), (T, S, int))
-        self.assertEqual(get_last_args(Callable[[], int]), (int,))
+        base_tp = Iterable[Tuple[T, S]]
+        try:
+            tp = base_tp[int, T]
+        except TypeError:
+            # python 3.5.3- - TypeError: Cannot change parameter count from 1 to 2
+            pass
```

Comment:
And `sys.version_info` here. Btw, it looks like some of these are repeated, so can be re-factored via a constant set at the top of the module.

Reply:
yes indeed